### ISSUE TITLE
求教timestep shift + sigmoid操作

### ISSUE THREAD

**Issue by zyx1213271098** [Akegarasu/lora-scripts#594]
作者你好，在timestep_samplingd的设置中，为什么要进行sigmoid操作？timesteps = logits_norm.sigmoid()，这样操作后timesteps值域[0.5, 1)，而sd3论文中timestep shift后值域是[0,1]。增加sigmoid操作是出于什么原因啊？

**Comment by zyx1213271098** [comment]
> 作者你好，在timestep_samplingd的设置中，为什么要进行sigmoid操作？timesteps = logits_norm.sigmoid()，这样操作后timesteps值域[0.5, 1)，而sd3论文中timestep shift后值域是[0,1]。增加sigmoid操作是出于什么原因啊？
sorry，我理解错了，torch.randn值域理论是负无穷到正无穷，原代码没有问题！
